# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/imhex/hotfix.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ImHex/lib/external/imgui/CMakeLists.txt.vanilla	2023-01-09 16:17:02.694171211 +0100
+++ ImHex/lib/external/imgui/CMakeLists.txt	2023-01-09 16:17:22.951171143 +0100
@@ -46,5 +46,5 @@
 if (WIN32)
     target_link_libraries(imgui PUBLIC Freetype::Freetype glfw3 opengl32.lib)
 elseif (UNIX)
-    target_link_libraries(imgui PUBLIC Freetype::Freetype glfw OpenGL::GL)
+    target_link_libraries(imgui PUBLIC Freetype::Freetype glfw3 OpenGL::GL)
 endif()
